docs/users/installation.md

```diff
@@ -75,6 +75,22 @@ following settings to `build.sbt`
  )
 ```
 
+> To enable SemanticDB related settings for all projects in your build.sbt, 
+> use the following setting which is available from Scalafix 0.9.14 and sbt 1.3.x . 
+> 
+> ```diff
+>  // build.sbt
+> inThisBuild(
+>  List(
+>    scalaVersion := "@SCALA212@", // @SCALA211@, or @SCALA213@
+> +  semanticdbEnabled := true,
+> +  semanticdbIncludeInJar := true,
```

Comment:
Let's recommend setting this to false by default.

Reply:
You're right.  Let's do it.
(I might have something wrong in my local project, which required `semanticdbIncludeInJar := true`  ;) )